purchases/src/main/kotlin/com/revenuecat/purchases/attributes/SpecialSubscriberAttributes.kt

```diff
@@ -0,0 +1,38 @@
+package com.revenuecat.purchases.attributes
+
+internal const val SPECIAL_KEY_EMAIL = "\$email"
+internal const val SPECIAL_KEY_DISPLAY_NAME = "\$displayName"
+internal const val SPECIAL_KEY_PHONE_NUMBER = "\$phoneNumber"
+internal const val SPECIAL_KEY_FCM_TOKENS = "\$fcmTokens"
+
+internal sealed class SubscriberAttributeKey(val serverValue: String) {
```

Comment:
maybe `reservedKey` is more intuitive? I got a bit confused when I read `serverValue` since it doesn't actually come from the server, it's more of an agreed-upon protocol between server and client

Reply:
There is a `SubscriberAttributeKey.Custom` which is used when someone passes something that is not one of the reserved ones. That's why I called it `serverValue`. Do you like it? I like that all types of keys are encapsulated in this class, but I agree the `serverValue` is confusing.

Reply:
I see. how about `backendKey`? the `value` part is what I found the most confusing at first because the entity is called `SubscriberAttributeKey`, so reading something called `value` I thought it'd mean the actual value instead of an alias for the key